CMakeLists.txt

```diff
@@ -63,42 +63,44 @@ find_library(LUA_LIBRARY lua)
 link_libraries(${LUA_LIBRARY})
 
```

Comment:
it might be nice, as part of this, to move the executable into a `samples` folder since it isn't going to be a shipping thing

Reply:
Well, this executable might not be - however we will need an executable build that auto starts Lua and any core dependencies such as CoreCLR (since Lua cannot start itself), though I guess that would be a separate project perhaps?

Reply:
To clarify - this would specifically be for deployments that cannot start themselves, such as Lua, if no "piloting" deployment/API is specified.

Reply:
Sure, but that sounds like something that belongs in a `tools` folder :)